src/dsp_tools/commands/start_stack.py

```diff
@@ -293,11 +286,17 @@ def _start_remaining_docker_containers(self) -> None:
         (Fuseki is already running at this point.)
         """
         if self.__stack_configuration.latest_dev_version:
-            subprocess.run(
-                "docker pull daschswiss/knora-api:latest".split(),
-                cwd=self.__docker_path_of_user,
-                check=True,
+            override_file = importlib.resources.files("dsp_tools").joinpath(
+                "resources/start-stack/docker-compose.override.yml"
             )
+            override_content = yaml.safe_load(override_file.read_bytes())
+            dockerhub_latest_tags = [x["image"] for x in override_content["services"].values()]
+            for tag in dockerhub_latest_tags:
+                subprocess.run(
+                    f"docker pull {tag}".split(),
+                    cwd=self.__docker_path_of_user,
+                    check=True,
+                )
```

Comment:
this could be simplified with `docker compose pull` which automatically calls `pull` on all components - and is much faster because it does so in parallel

Reply:
Thanks, that's indeed a huge improvement.